tests/dCommonTests/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ list(APPEND DCOMMONTEST_SOURCES ${DENUMS_TESTS})
 add_executable(dCommonTests ${DCOMMONTEST_SOURCES})
 
 # Link needed libraries
-target_link_libraries(dCommonTests ${COMMON_LIBRARIES} GTest::gtest_main)
+target_link_libraries(dCommonTests ${COMMON_LIBRARIES} dGame GTest::gtest_main)
```

Comment:
Is this addition necessary?  dCommonTests shouldn't need anything aside from common libs

Reply:
Generally yes, you are meant to attach headers to INTERFACE_INCLUDE_DIRECTORIES of a target and pull that in as a link library which automatically adds the include search path and there is at least one test that included dGame headers.

I'm open for suggestions though, would just like to keep the cpp/h files unchanged in these PRs

Reply:
I see what you are talking about with one test using those headers.  Those can be safely removed.  They were a result of copy and paste from a previous test and are fully unneeded for common tests as we dont even link with dGame previously, so we couldn't even use the libraries from there anyways.  I think its better to just remove the two offending headers than to add a full library to link against, but its up to you.

Reply:
i just put up a pr to resolve this issue.  This link should not be necessary anymore.

Reply:
Sure. I'll rebase when this one is in and remove the link